presto-hive/src/test/java/com/facebook/presto/hive/TestHiveFileFormats.java

```diff
@@ -381,7 +381,7 @@ public void testFailForLongVarcharPartitionColumn()
                 .isFailingForRecordCursor(new ParquetRecordCursorProvider(false, HDFS_ENVIRONMENT), expectedErrorCode, expectedMessage);
 
         TestingConnectorSession session = new TestingConnectorSession(
-                new HiveSessionProperties(new HiveClientConfig().setParquetOptimizedReaderEnabled(true)).getSessionProperties());
+                new HiveSessionProperties(new HiveClientConfig().setParquetOptimizedReaderEnabled(true).setParquetPredicatePushdownEnabled(true)).getSessionProperties());
```

Comment:
Instead of enabling parquet predicate pushdown only in `testTruncateVarcharColumn` and `testFailForLongVarcharPartitionColumn` how about we add another boolean parameter (`predicatePushdownEnabled`) to the existing `testParquet()` to test for both enabled/disabled?